app/views/admin/organisations/_people_order.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= render "govuk_publishing_components/components/summary_list", {
+  title:,
+  edit: {
+    href: "#",
+    link_text: sanitize("Reorder <span class=\"govuk-visually-hidden\">#{title}</span>"),
+    link_text_no_enhance: true
+  }
+} do %>
+  <% organisation_roles.each do |organisation_role| %>
+    <%= render "components/summary_card_component", {
+      title: organisation_role.role.name,
+      summary_card_actions: [
+        {
+          label: "Edit",
+          href: edit_admin_role_path(organisation_role.role)
+        }
+      ],
+      rows: [
+        *(if organisation_role.role.current_person
+            [{
+               key: "Occupant",
+               value: organisation_role.role.current_person_name,
+               actions: [{
+                           label: "Edit",
+                           href: edit_admin_person_path(organisation_role.role.current_person)
+                         },]
+             }]
+            else
+              [{
+                 key: "Occupant",
+                 value: "Vacant",
+               }]
+            end),
```

Comment:
This is using a splat operator which removes the hash from the array. i.e. *["a] == "a". 

It can also work inversely i.e. *"a" == ["a"]

I think you can probably simplify this a bit by just removing the array from the conditional and  removing the splat.

```
  rows: [
        if organisation_role.role.current_person
          {
            key: "Occupant",
            value: organisation_role.role.current_person_name,
            actions: [
              {
                label: "Edit",
                href: edit_admin_person_path(organisation_role.role.current_person)
              },
            ]
          }
        else
          {
            key: "Occupant",
            value: "Vacant",
          }
        end,
      ]
```

Reply:
Cool, thanks @davidgisbey. Yeah, got a bit mixed up with the necessity of the splat operator there.